src/main/java/MysqlDAO.java

```diff
@@ -62,6 +60,50 @@
 		return getList;
 	}
 
+	protected Boolean updateQuery(String sql_query) {
```

Comment:
I wonder if this would be a good opportunity to get rid of some code reuse between this and the createQuery method for future refactoring.

Reply:
Yeah, the primary things duplicated here are just the opening of connections and cleanup. Executing the statements differs only in return type. Not sure how I feel passing around open connections yet or if I just want to utilize a pool. Hmm..